silk-core/src/main/scala/org/silkframework/dataset/rdf/GraphStoreFileUploadTable.scala

```diff
@@ -0,0 +1,19 @@
+package org.silkframework.dataset.rdf
+
+import org.silkframework.config.{Task, TaskSpec}
+import org.silkframework.entity.{Entity, EntitySchema}
+import org.silkframework.execution.local.{EmptyEntityTable, LocalEntities}
+import org.silkframework.runtime.resource.FileResource
+
+/** A number of files that should be uploaded via the GraphStore protocol. */
+trait GraphStoreFileUploadTable {
+  def files: Traversable[FileResource]
+}
+
+case class LocalGraphStoreFileUploadTable(files: Traversable[FileResource], task: Task[TaskSpec]) extends LocalEntities with GraphStoreFileUploadTable {
+  override def entitySchema: EntitySchema = EntitySchema.empty
+
+  override def entities: Traversable[Entity] = Traversable.empty
+
+  override def updateEntities(newEntities: Traversable[Entity]): LocalEntities = EmptyEntityTable(task)
```

Comment:
@andreas-schultz Here was a small merge issue you may want to take a look at: the update method.

Reply:
It should just return itself. Updating entities has no effect since the entities are always empty for this table.